llvm/test/CodeGen/PowerPC/scalar-min-max.ll

```diff
@@ -117,13 +117,12 @@ define dso_local float @testfmax_fast(float %a, float %b) local_unnamed_addr {
 ;
 ; NO-FAST-P9-LABEL: testfmax_fast:
 ; NO-FAST-P9:       # %bb.0: # %entry
-; NO-FAST-P9-NEXT:    xsmaxcdp f1, f1, f2
+; NO-FAST-P9-NEXT:    xsmaxdp f1, f1, f2
```

Comment:
The only test changes are on these fcmp + select patterns. This should have test coverage on the direct intrinsic uses